package.json

```diff
@@ -71,12 +71,14 @@
     "typescript": "5.7.3",
     "vite": "5.4.12",
     "vite-plugin-node-polyfills": "0.23.0",
-    "vitest": "2.1.9"
+    "vitest": "2.1.9",
+    "@synthetixio/synpress-metamask": "0.0.11"
```

Comment:
I remember I've deleted this MM tests yesterday, maybe I've forgot this package. Can you double check if this is really needed? 🙏🏽